public/js/routes.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+var app = app || {};
+
+page('/', () => { /index });
+
+page('/about', () => {
+  $('.data-content').hide();
+  $('#about').fadeIn();
+});
+page('/interests', () => {
+  $('.data-content').hide();
+  $('#interests').fadeIn();
+});
+page('/projects', () => {
+  $('.data-content').hide();
+  $('#projects').fadeIn();
+});
+page('*', () => {
+  $('.data-content').hide();
+  $('#projects').fadeIn();
+});
```

Comment:
You do not need a catch-all route here since that is already being handled by your server.

Reply:
okay